boards/arm/frdm_k64f/frdm_k64f.dts

```diff
@@ -156,7 +156,6 @@ arduino_spi: &spi0 {
 		compatible = "zephyr,sdhc-spi-slot";
 		reg = <0>;
 		status = "okay";
-		label = "SDHC_0";
 		mmc {
```

Comment:
Hi @hunterhu3000 , suggest to put this fix in another commit, or even another PR.

Reply:
got it. Thanks
A new PR without this change is submitted. Thanks to review.